eq-author/src/App/publish/updatePublishDetails.graphql

```diff
@@ -0,0 +1,10 @@
+  mutation UpdatePublishDetails($input: UpdatePublishDetailsInput!) {
+    updatePublishDetails(input: $input) {
+      id
+      publishDetails {
+        formType
+        surveyId
+      }
+    }
+  }
+  
```

Comment:
This whole mutation has an extra indentation, which is probably why it doesnt have a newline at the end